mne/source_estimate.py

```diff
@@ -1696,6 +1771,127 @@ def get_peak(self, tmin=None, tmax=None, mode='abs',
                 time_idx if time_as_index else self.times[time_idx])
 
 
+class VectorSourceEstimate(SourceEstimate):
+    """Container for full MNE surface estimate, including dipole directions.
+
+    Parameters
+    ----------
+    data : array of shape (n_dipoles, 3, n_times)
+        The data in source space. Each dipole contains three vectors that
+        denote the dipole strength in X, Y and Z directions over time.
+    vertices : array | list of two arrays
+        Vertex numbers corresponding to the data.
+    tmin : float
+        Time point of the first sample in data.
+    tstep : float
+        Time step between successive samples in data.
+    subject : str | None
+        The subject name. While not necessary, it is safer to set the
+        subject parameter to avoid analysis errors.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    Attributes
+    ----------
+    subject : str | None
+        The subject name.
+    times : array of shape (n_times,)
+        The time vector.
+    shape : tuple
+        The shape of the data. A tuple of int (n_dipoles, n_times).
+    Notes
```

Comment:
you need a newline before `Notes`, this will not render properly

Can you check CircleCI output for all classes and examples to verify they render properly?